bot/src/discord_commands/jibiki_all_search.js

```diff
@@ -0,0 +1,254 @@
+const {throwPublicErrorInfo} = require('./../common/util/errors.js');
+const {Navigation, NavigationChapter} = require('monochrome-bot');
+const axios = require('axios');
+const constants = require('./../common/constants.js');
+
+const jibikiApiUri = 'https://api.jibiki.app';
+const maxFieldsPerPage = 8;
+
+module.exports = {
+  commandAliases: ['jibiki', 'jb'],
+  cooldown: 3,
+  uniqueId: 'best_dictionary',
+  shortDescription: 'Search Jibiki.',
+  usageExample: '<prefix>jibiki 女らしい',
+  async action(bot, msg, suffix, monochrome) {
+    if (!suffix) {
+      const {prefix} = msg;
+      return throwPublicErrorInfo('jibiki', `Say **${prefix}jibiki [text]** to search for words. For example: **${prefix}jibiki dance**`, 'No suffix');
+    }
+
+    const response = await axios.get(`${jibikiApiUri}/all?query=${encodeURIComponent(suffix)}&sentenceCount=5`);
+
+    if (response.status === 200) {
+      const wordPages = [];
+      const kanjiPages = [];
+      const sentencePages = [];
+
+      for (let page = 0; page < Math.ceil(response.data.length / maxFieldsPerPage); page += 1) {
+        const wordFields = [];
+
+        for (
+          let field = 0;
+          field < response.data.length - (page * maxFieldsPerPage);
+          field += 1
+        ) {
+          const entry = response.data[(page * maxFieldsPerPage) + field];
+          let word = '';
+
+          if (entry.word.forms[0].kanji.info !== null) {
+            word += `*Kanji note; ${entry.word.forms[0].kanji.info}*\n`;
+          }
+          if (entry.word.forms[0].reading.info !== null) {
+            word += `*Reading note; ${entry.word.forms[0].reading.info}*\n`;
+          }
+
+          if (entry.word.jlpt !== null) {
+            word += `**JLPT N${entry.word.jlpt}**\n`;
```

Comment:
Suggestion: using bold for this looks a bit heavy to me. You might consider instead putting it in a code block, like 

```js
word += `\`JLPT N${entry.word.jlpt}\`\n`;
```

That makes it look like this

![image](https://user-images.githubusercontent.com/4397296/73586027-ec078680-4475-11ea-9238-bb5e2998cf30.png)